docs: :pencil2: simple edits during review of design docs (#255)
seedcase-project/check-datapackage: docs/design/interface.qmd

## Flow

This is the flow of how `check-datapackage` is used, or at least how
properties flow through `check-datapackage` functions and classes.

```{mermaid}
%%| label: fig-interface-flow

> %%| fig-cap: "Flow of the input and output into functions and classes when using `check-datapackage`."

Reviewer: ```suggestion
%%| fig-cap: "Flow of inputs and outputs through the functions and classes of `check-datapackage`."
```
